topologytest/hlv_test.go

```diff
@@ -73,12 +133,41 @@ func TestHLVCreateDocumentSingleActor(t *testing.T) {
 			peers, _ := setupTests(t, tc.topology, tc.activePeerID)
 
 			docBody := []byte(fmt.Sprintf(`{"peer": "%s", "topology": "%s"}`, tc.activePeerID, tc.description()))
-			docVersion := peers[tc.activePeerID].CreateDocument(tc.collectionName(), tc.docID(), docBody)
+			docVersion := peers[tc.activePeerID].CreateDocument(tc.collectionName(), tc.docID(tc.activePeerID), docBody)
 			waitForVersionAndBody(t, tc, peers, docVersion, docBody)
 		})
 	}
 }
 
+func TestHLVCreateDocumentMultiActor(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyCRUD, base.KeyImport, base.KeyVV)
+
+	for _, tc := range getMultiActorTestCases() {
+		t.Run(tc.description(), func(t *testing.T) {
+			peers, _ := setupTests(t, tc.topology, "")
+
+			var docVersionList []BodyAndVersion
+			peerList := tc.PeerNames()
+
+			// grab sorted peer list and create a list to store expected version,
+			// doc body
+			for _, peerName := range peerList {
```

Comment:
readability, the shorter the test code the easier to read IMO

```suggestion
			for peerName := range tc.PeerNames() {
```